zilliqa/src/transaction.rs

```diff
@@ -914,6 +927,12 @@ impl FromStr for ZilAmount {
     }
 }
 
+impl From<crate::cfg::Amount> for ZilAmount {
```

Comment:
We deliberately didn't include this to force the caller to use `from_amount`, where they will hopefully acknowledge the caveats around rounding.